lib/acts_as_taggable_on_mongoid/taggable/list_tags.rb

```diff
@@ -18,7 +24,36 @@ def tag_definition(tag_type)
         tag_types[tag_type] ||= ActsAsTaggableOnMongoid::Taggable::TagTypeDefinition.new(self, tag_type)
       end
 
+      # rubocop:disable Metrics/BlockLength
+
       class_methods do
+        def tag_types
+          self.my_tag_types = cleanup_tag_types(my_tag_types, self)
+
+          my_tag_types
+        end
+
+        def cleanup_tag_types(tag_types, klass)
```

Comment:
Doesn't depend on instance state (maybe move it to another class?) (UtilityFunction)